### DISCUSSION TITLE
disable cruise control deployment

### DISCUSSION THREAD

**Discussion by ghost** [strimzi/strimzi-kafka-operator#9652] [Q&A]
Is it possible to disable the cruise control deployment in strimzi kafka ?

**Comment by scholzj** [comment]
You can remove the `cruiseControl` section from the Kafka custom resource.

**Comment by dberardo-com** [comment]
is it possible to simply keep it defined but then seta replica config to 0 ? it would be great IMO

**Reply by scholzj** [reply]
No, that is not possible.
